apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/block-inserter/index.js

```diff
@@ -0,0 +1,47 @@
+/* global fullSiteEditing */
+
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { render } from '@wordpress/element';
+import { Inserter } from '@wordpress/editor';
+import { compose } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
+
+const PostContentBlockAppender = compose(
```

Comment:
We should separate this into its own component file.

Reply:
Moved in a1e8aed.